{"name": "test_with_invalid_email", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55705e9ee013 <unknown>\n#1 0x55705e750a08 <unknown>\n#2 0x55705e786fb6 <unknown>\n#3 0x55705e787171 <unknown>\n#4 0x55705e7b9dd4 <unknown>\n#5 0x55705e7a495d <unknown>\n#6 0x55705e7b7b94 <unknown>\n#7 0x55705e7a4823 <unknown>\n#8 0x55705e77a08a <unknown>\n#9 0x55705e77b1e5 <unknown>\n#10 0x55705ea1be5d <unknown>\n#11 0x55705ea35061 <unknown>\n#12 0x55705ea1dbb5 <unknown>\n#13 0x55705ea35e08 <unknown>\n#14 0x55705ea112f0 <unknown>\n#15 0x55705ea51988 <unknown>\n#16 0x55705ea51b08 <unknown>\n#17 0x55705ea6b5dd <unknown>\n#18 0x7ff24805d609 <unknown>", "trace": "request = <SubRequest 'setup_browser' for <Function test_with_invalid_email>>\n\n    @pytest.fixture(scope=\"function\", autouse=True)\n    def setup_browser(request):\n        browser_version = request.config.getoption('--browser_version')\n        browser_version = browser_version if browser_version != \"\" else DEFAULT_BROWSER_VERSION\n        options = Options()\n        selenoid_capabilities = {\n            \"browserName\": \"chrome\",\n            \"browserVersion\": browser_version,\n            \"selenoid:options\": {\n                \"enableVNC\": True,\n                \"enableVideo\": True,\n            }\n        }\n        options.capabilities.update(selenoid_capabilities)\n        login = os.getenv('LOGIN')\n        password = os.getenv('PASSWORD')\n        driver = webdriver.Remote(\n            command_executor=f'https://{login}:{password}@selenoid.autotests.cloud/wd/hub',\n            options=options\n        )\n        browser.config.driver = driver\n    \n        browser.config.timeout = 60\n        browser.config.window_width = 1200\n        browser.config.window_height = 800\n        browser.config.base_url = 'https://www.workingnomads.com'\n        cookie_notification_text = \"We value your privacy\"\n    \n        # Find the element that contains the notification text\n>       notification_element = WebDriverWait(driver, 10).until(\n            EC.presence_of_element_located((By.XPATH, f\"//*[contains(text(), '{cookie_notification_text}')]\"))\n        )\n\ntests\\conftest.py:70: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6fef47089370c4e29879d5a574bf6a94\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x00000152904ED800>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55705e9ee013 <unknown>\nE       #1 0x55705e750a08 <unknown>\nE       #2 0x55705e786fb6 <unknown>\nE       #3 0x55705e787171 <unknown>\nE       #4 0x55705e7b9dd4 <unknown>\nE       #5 0x55705e7a495d <unknown>\nE       #6 0x55705e7b7b94 <unknown>\nE       #7 0x55705e7a4823 <unknown>\nE       #8 0x55705e77a08a <unknown>\nE       #9 0x55705e77b1e5 <unknown>\nE       #10 0x55705ea1be5d <unknown>\nE       #11 0x55705ea35061 <unknown>\nE       #12 0x55705ea1dbb5 <unknown>\nE       #13 0x55705ea35e08 <unknown>\nE       #14 0x55705ea112f0 <unknown>\nE       #15 0x55705ea51988 <unknown>\nE       #16 0x55705ea51b08 <unknown>\nE       #17 0x55705ea6b5dd <unknown>\nE       #18 0x7ff24805d609 <unknown>\n\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException"}, "start": 1689795548330, "stop": 1689795548330, "uuid": "8fd01822-a052-4904-800c-ff7d08f3c2db", "historyId": "fd9930532373ce8080ff335f6c309eaf", "testCaseId": "267ad1994ab712361545180c4a21d3b1", "fullName": "tests.test_signup#test_with_invalid_email", "labels": [{"name": "feature", "value": "Signing up with invalid credentials"}, {"name": "story", "value": "Verify that an appropriate error message is displayed and still on same url"}, {"name": "tag", "value": "web"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_signup"}, {"name": "host", "value": "LAPTOP-OFQSKCIR"}, {"name": "thread", "value": "19348-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_signup"}]}